contactform.php

```diff
@@ -567,12 +567,13 @@ public function sendMessage()
             && empty($mailAlreadySend)
             && ($sendConfirmationEmail || $sendNotificationEmail)
         ) {
+            $message = version_compare(_PS_VERSION_, '8.0.0', '>=') ? stripslashes($message) : Tools::stripslashes($message);
```

Comment:
As this PR targets `develop` branch, I think that you can use built-in `stripslashes()` and remove the check on PrestaShop version.

Reply:
I think `dev` branch in this repo is different from `develop` branch in Core repo :)
`develop` branch in Core repo means PS 9 for now, but `dev` branch in this repo support PS 1.7.2+

Reply:
Oh my bad, I thought I was on prestashop repo.